### DISCUSSION TITLE
Can't write bangla in any browser using Open Bangla Keyboard in Kali Linux

### DISCUSSION THREAD

**Discussion by zawad1804** [OpenBangla/OpenBangla-Keyboard#304] [Q&A]
OBK is not working in Chrome or Firefox browser. My desktop environment is GNOME 42.2, OS: Kali Linux, Windowing System: X11.

**Comment by mominul** [comment]
Can you try running firefox by using this command?
```
GTK_IM_MODULE=ibus firefox
```

**Reply by ahmubashshir** [reply]
`env GTK_IM_MODULE=ibus QT_IM_MODULE=ibus XMODIFIERS=@im=ibus firefox`

**Reply by zawad1804** [reply]
Didn't work for me 😭😭 ki je peray asi bangla typing niye :)

**Comment by zawad1804** [comment]
Using that command didn't work for me :( 
![image](https://user-images.githubusercontent.com/74665021/178068378-b09acbdd-e3bc-48c0-a285-2882ad92b1ac.png)

**Reply by mominul** [reply]
আপনি টেক্সট এডিটরে বাংলা লিখতে পারছেন?

**Reply by zawad1804** [reply]
জ্বী, টেক্সট এডিটর, টেলিগ্রাম সফটওয়্যার, ওভারভিউ এর সার্চ বারেও বাংলা লিখতে পারছি, শুধু ব্রাউজারেই কাজ করেনা

**Reply by ahmubashshir** [reply]
try with a new firefox profile, I had somewhat similar problem, but after nuking the profile, it was fixed.
